solutions/cf-20C.mdx

```diff
@@ -76,4 +76,57 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+	
+```py
+
+import heapq
+import sys
+
+
+def dijkstra(graph, source):
```

Comment:
type annotations

Reply:
not always needed but in this case yeah.

Reply:
I've tried giving annotations for the function
It works fine in my PyCharm IDE
But when I try to submit on codeforces, its giving a runtime error
<img width="788" alt="image" src="https://user-images.githubusercontent.com/78244925/175804778-4fee08fc-aa99-45be-baa0-7acf7d96bb1e.png">
Could you please suggest what to do?